app/controllers/posts_controller.rb

```diff
@@ -2,10 +2,30 @@ class PostsController < ApplicationController
   layout 'application'
 
   def index
-    # Code for the index method
+    @user = User.find(params[:user_id])
+    @posts = @user.posts
```

Comment:
- [ ] It would be better if we fix the N+1 query problem when fetching users' posts and comments as we can see in the screenshot below
![Screenshot from 2023-10-05 09-00-24](https://github.com/yemidada/blog-app/assets/46388113/50bd54e9-34f5-4961-929b-cba939edddd3)
Here is how we can fix that `@posts = @user.posts.includes(:comments)`